### DISCUSSION TITLE
How many themes does cCart have to use?

### DISCUSSION THREAD

**Discussion by dnationsoftbd5** [imranertaza/cCart#148] [Q&A] [answered]
Hi
Could anybody please tell me how many theme does cCart have to use?

Thanks in advance

**Comment by imranertaza** [comment, marked as answer]
Hi
Yes, you will be happy to know that there are 3 themes are already developed in cCart to use.

1) For gadget products 
2) For fashion products
3) Another one for fashion products

Thank you
